modules/ROOT/pages/logging-in-mule.adoc

```diff
@@ -24,12 +24,27 @@ You can also view the app log in this default location, unless the log file path
 
 == Runtime Log
 
-The runtime log (`mule_ee.log`) contains information about app and lifecycle events. For example, the log records an entry when a Mule service or app starts, deploys, stops, or undeploys. The log is located in the `log4j2.xml` file in the `conf` directory. You can customize this file when running the server in standalone mode.
+The runtime log (`mule_ee.log`) contains information about app and lifecycle events. For example, the log records an entry when a Mule service or app starts, deploys, stops, or undeploys.
 
-To view the runtime log in Anypoint Studio:
+The runtime log configuration is located in the `log4j2.xml` file, in the `/conf` directory. You can customize this file when running the server in standalone mode.
+
+=== Change the Log Level
+
+You can change the runtime log level by modifying the `level` value of the following packages in the `log4j2.xml` file:
+
+[source,xml,linenums]
+----
+  <AsyncLogger name="org.mule" level="INFO"/>
+  <AsyncLogger name="com.mulesoft" level="INFO"/>
+----
+Accepted values are: `DEBUG`, `ERROR`, `INFO`, `TRACE`, and `WARN`.
+
+To change the log level only for specific Connectors, see xref:enable-verbose-logging.adoc[Enable Verbose Logging].
+
+=== View the Runtime Log in Anypoint Studio:
+
+Click *Anypoint Studio* -> *About Anypoint Studio* -> *Installation Details* -> *Configuration* tab -> *View Error Log*:
```

Comment:
Replace -> with just >

I've also seen writers put the whole string in one set of *:  "Click *Anypoint Studio > About Anypoint Studio > Installation...*."